examples/planout-example/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "mendel-full-example",
+  "version": "1.1.0",
+  "description": "Example of app using mendel, es6, jsx, react, planout for assignment, server-side-rendering, multivariant and multilayer support.",
+  "main": "index.js",
+  "scripts": {
+    "test": "eslint .",
+    "build": "mendel",
+    "development": "nodemon index.js",
+    "production": "NODE_ENV=production node index.js"
+  },
+  "dependencies": {
+    "cookie-parser": "^1.4.3",
+    "express": "^4.13.4",
+    "mendel-middleware": "*",
+    "morgan": "^1.7.0",
+    "node-uuid": "^1.4.7",
+    "planout": "^3.0.1",
+    "react": "^0.14.8",
+    "react-dom": "^0.14.8"
+  },
+  "devDependencies": {
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-react": "^6.5.0",
+    "babelify": "^7.3.0",
+    "eslint-plugin-react": "^5.2.2",
+    "jasmine-core": "^2.5.2",
+    "karma": "^1.3.0",
+    "karma-browserify": "^5.1.0",
+    "karma-jasmine": "^1.0.2",
+    "karma-jasmine-diff-reporter": "^0.6.2",
+    "karma-phantomjs-launcher": "^1.0.2",
+    "karma-spec-reporter": "0.0.26",
+    "mendel": "*",
+    "mendel-development-middleware": "*",
+    "mendel-manifest-extract-bundles": "*",
+    "nodemon": "^1.9.2",
+    "react-addons-test-utils": "^0.14.8",
+    "watchify": "^3.7.0"
+  },
+  "babel": {
+    "presets": [
+      "es2015",
+      "react"
+    ],
+    "retainLines": true
+  },
+  "contributors": [
+    {
+      "name": "Irae Carvalho",
+      "email": "irae@irae.pro.br"
+    },
+    {
+      "name": "David Gomez",
+      "email": "david.garcia.mx@gmail.com"
+    },
+    {
+      "name": "Shalom Volchok",
+      "email": "shalom@digitaloptgroup.com"
+    }
```

Comment:
Feel free to have only yourself on the list if you want, or maybe move yourself to the top if you wish 😉 
Also, this domain don't serve any webpages, so you might want to double-check for typos, or maybe you only use it for email?